src/components/action-bar/action-bar.tsx

```diff
@@ -0,0 +1,262 @@
+import {
+    Component,
+    Event,
+    EventEmitter,
+    h,
+    Host,
+    Prop,
+    State,
+    Element,
+} from '@stencil/core';
+import { ActionBarItem } from './action-bar.types';
+import { MenuItem, OpenDirection } from '../menu/menu.types';
+import { ListSeparator } from '../list/list-item.types';
+
+/**
+ * An action bar is a user interface element commonly found in software applications and websites.
+ * It typically appears at the top of the screen or within a specific section
+ * and serves as a centralized hub for accessing various actions and commands
+ * relevant to the current context or page.
+ *
+ * The action bar often contains a set of clickable icons or buttons (icons + labels)
+ * that represent specific actions, such as saving, deleting, editing, sharing,
+ * or bulk operations for selected items.
+ *
+ * The purpose of an action bar is to provide quick and convenient access to
+ * frequently used functionalities, enabling users to perform common tasks efficiently.
+ * It enhances usability by organizing important actions in a visually prominent and easily accessible location.
+ *
+ * The action bar's design and layout can vary based on the platform or application,
+ * but its primary goal remains consistent—to
+ * empower users to interact with the software and perform desired actions effortlessly.
+ * @exampleComponent limel-example-action-bar
+ * @exampleComponent limel-example-action-bar-overflow-menu
+ * @exampleComponent limel-example-action-bar-colors
+ * @exampleComponent limel-example-action-bar-floating
+ * @exampleComponent limel-example-action-bar-styling
+ * @exampleComponent limel-example-action-bar-as-primary-component
+ */
+@Component({
+    tag: 'limel-action-bar',
+    shadow: true,
+    styleUrl: 'action-bar.scss',
+})
+export class ActionBar {
+    /**
+     * Items that are placed in the action bar.
+     * These represent primary actions.
+     */
+    @Prop()
+    public actionBarItems: Array<ActionBarItem | ListSeparator> = [];
```

Comment:
I propose changing this to just `items`. Short and concise names are usually better, and the extra `actionBar` prefix does not add anything in this scenario. There should be no need to repeat the name of the component for this property

Reply:
Another option is to name it `actions`

Reply:
since there are dividers (`ListSeparator`) in there too, maybe `items` is still a better name?